_posts/en/newsletters/2019-09-25-newsletter.md

```diff
@@ -46,12 +46,12 @@ popular Bitcoin infrastructure projects.
     He describes some of their findings and advocates the following rule
     changes:
 
-    > * Replace the separate sigops counter with a "executed sigops must
+    > * Replace the separate sigops counter with a [rule that] "executed sigops must
     >   not exceed (witness size / [12.5 vbytes]) + 1" rule (already in the BIP).
```

Comment:
@harding should now drop the second instance of "rule" in the sentence

Reply:
Ugh.  I'm going to revert this to just sipa's text, I think it's good enough and anyone who's confused can just read his entire email.